infrastructure/modules/products/pars.tf

```diff
@@ -18,7 +18,7 @@ resource "azurerm_storage_account" "pars" {
 
 resource "azurerm_cdn_profile" "pars" {
   name                = "mhrapars${var.environment}"
-  location            = var.location
+  location            = "westeurope"
```

Comment:
not allowed CDN in uk south so needed to force to westeurope for non prod / prod (dev currently has default region as westeurope)